xml_converter/proto/waypoint.proto

```diff
@@ -38,7 +38,7 @@ message Icon {
     bool scale_on_map_with_zoom = 23;
     string tip_description = 24;
     string tip_name = 25;
-    RGBA rgba = 26;    
+    RGBAColor rgba_color = 26;    
```

Comment:
There is already a class in Godot called Color and it was interfering with the proto functions so it had to be renamed. 

I think `RGBA_Color` is our best option since it retains what the field is used for but also specify what data it contains